scripts/docker/build_docker.py

```diff
@@ -1,471 +1,377 @@
+#!/usr/bin/env python
+
 import argparse
+import sys
 import os
 import os.path
 import time
 import tempfile
 import shutil
+import pprint
+# noinspection PyPackageRequirements
```

Comment:
This script has a problem right now in that it hangs out in the project along-side a real python module (svtk). So pycharm assumes it must have the same package requirements. I decided against trying to merge this into svtk because it would change its location and behavior too much. I feel like it should eventually be merged though.

Reply:
That makes sense, thank you!